main.go

```diff
@@ -113,10 +113,15 @@ func NewGame(g *Game) {
 	g.Sprites[spritePlayer] = loadSprite("Player")
 	g.Sprites[spriteZombie] = loadSprite("Zombie")
 
+	// Load entities from map
+	entities := level.LayerByIdentifier("Entities")
+
 	// Add player to the game
+	playerPosition := entities.EntityByIdentifier("Player").Position
+
```

Comment:
I'd omit this white space because both this chunk of code above and below it are about the same logical thing, but this is such a nitpick that you can straight up ignore it, just sharing my style/thinking :)